src/imputors/locf.jl

```diff
@@ -15,13 +20,15 @@ that all missing values will be imputed.
 
 ```
 """
-function impute!(imp::LOCF, ctx::Context, data::AbstractVector)
-    start_idx = findfirst(ctx, data) + 1
-    for i in start_idx:length(data)
-        if ismissing(ctx, data[i])
-            data[i] = data[i-1]
+function impute!(imp::LOCF, data::AbstractVector)
+    imp.context() do c
+        start_idx = findfirst(c, data) + 1
+        for i in start_idx:length(data)
+            if ismissing(c, data[i])
+                data[i] = data[i-1]
```

Comment:
Do you think it is worth:
```suggestion
                @inbounds data[i] = data[i-1]
```

Reply:
Maybe we should make an issue to add `@inbounds` where appropriate throughout the codebase?

Reply:
maybe.
It is always hard to know when it would help

Reply:
https://github.com/invenia/Impute.jl/issues/25